### DISCUSSION TITLE
Make profile description section fixed width.

### DISCUSSION THREAD

**Discussion by mohitkyadav** [community/community#18802] [Other Feature Feedback, Questions, & Ideas]
The description section on GitHub is responsive 💯 . I added this description to my profile.
As you can see on profile page it resembles a `04` but when viewed from a large screens or via mini profile (on hover) it loses its shape. I have also tried by adding new lines in between each line, but that also does not work.

![image](https://user-images.githubusercontent.com/25580776/174304890-5f8c14e5-237d-4cb6-a684-85670c955f64.png)

This is how it looks on larger screen and mini profile.

![image](https://user-images.githubusercontent.com/25580776/174305006-1e212d92-bf7e-455f-8048-e79828812d04.png)

Let me know what you think about this suggestion.

**Comment by Or1onn** [comment]
You should edit your README to fit your mini profile and not widescreen

**Reply by mohitkyadav** [reply]
@Or1onn it's not about readme, please give the question another look. I'm referring to the "description" section on your profile.

**Reply by Or1onn** [reply]
You need to switch to a new line (ENTER) in your profile yourself, there are no other options

**Reply by mohitkyadav** [reply]
I have tried that, it trims all the new lines. I am aware there is know way of doing that, that is why this discussion exist in the first place.
